stages/org.osbuild.kernel-cmdline.meta.json

```diff
@@ -30,6 +30,12 @@
         "description": "Additional kernel boot options",
         "type": "string",
         "default": ""
+      },
+      "kernel_line_size": {
```

Comment:
Maybe `kernel_cmdline_size` here? it seems `cmdline` is slightly more descriptive than `line` (?)